contribution/vagrant/Vagrantfile

```diff
@@ -109,6 +112,14 @@ Vagrant.configure("2") do |config|
     config.vm.provision :shell, path: kubearmor_home + "/contribution/self-managed-k8s-selinux/enable_selinux.sh"
 
   else # ubuntu
+    if NETNEXT == "0" or NETNEXT == "-1" then
+        # Updating kernel image to 5.15 required for building latest version of bpftool.
+        config.vm.provision :shell, :inline => "sudo apt-get update -y"
+        config.vm.provision :shell, :inline => "sudo apt-get install linux-image-5.15.0-73-generic linux-headers-5.15.0-73-generic linux-tools-generic -y"
+        config.vm.provision :shell, :inline => "sudo sed -i 's/^GRUB_DEFAULT=.*/GRUB_DEFAULT=\"Advanced options for Ubuntu>Ubuntu, with Linux 5.15.0-73.80~20.04.1 \"/' /etc/default/grub"
+        config.vm.provision :shell, :inline => "sudo update-grub"
+        config.vm.provision :reload
+    end
```

Comment:
We shouldn't do this

Reply:
Thanks a lot for the quick and clear review @daemon1024 , Repo said linux-genric-tools >=1.15 is required for building bpftool , However works fine for the static binaries of ubuntu 18,20,22